requirements.txt

```diff
@@ -16,6 +16,7 @@ mlx>=0.5.1,<0.6.0; sys_platform == 'darwin' and platform_machine == 'arm64'
 # HabanaLabs / Intel Gaudi env comes with Python 3.10 and slightly older
 # versions of some dependencies. Use '3.10' as an indicator.
 # Habana installer has NumPy 1.23.5
+munch
```

Comment:
can we pin this dep?

Reply:
I've pinned. But why it is important?

Reply:
The repo is following this practice.
One example is future-proofing, It allows us to gradually adopt new features and improvements from the dependency, making it easier to stay current and reducing the effort required for major upgrades. This was we can avoid pulling a recent version that would break backward compatibility.